File: Jules-workspace/screenshot.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('file:///app/Jules-workspace/pages/17.0_nXX_الْجُمْلَةُ الِاسْمِيَّةُ (الْمُبْتَدَأُ وَالْخَبَرُ) وَإِعْرَابُهُمَا (تابع).html');
  await page.evaluate(() => {
      const link = document.createElement('link');
      link.rel = 'stylesheet';
      link.href = 'file:///app/Jules-workspace/styles/main.css';
      document.head.appendChild(link);
  });
  await page.waitForTimeout(1000);
  await page.screenshot({ path: '/app/Jules-workspace/screenshot.png', fullPage: true });
  await browser.close();
})();

File: screenshot.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page(
        record_video_dir="videos/"
    )
    page.goto("file:///app/pages/page_124_h4pom.html")
    page.wait_for_timeout(2000)
    page.screenshot(path="screenshot.png", full_page=True)
    video_path = page.video.path()
    print(video_path)
    browser.close()
